api/config/services.yaml

```diff
@@ -131,6 +136,13 @@ services:
             - "%env(FRONTEND_BASE_URL)%"
             - "%env(MAIL_FROM)%"
 
+    App\Service\TranslateToAllLocalesService:
+        public: true
+        arguments:
+            - '@translator'
+            - '@translator'
```

Comment:
Is twice the same argument really correct here? Does that take the type annotation into account in addition to the alias which you write here?

Reply:
Symfony translations does not expose the TranslatorBagInterface as service, but the implementation of TranslatorInterface offered by Symfony translations also implements that interface.
Because Union Types did not work well with the DI last time, i did not try it this time.
Yes, it's on purpose. The same class, but as the implementation of 2 different interfaces.